### ISSUE TITLE
[A11Y]: Accessibility audit round 5 issues

### ISSUE THREAD

**Issue by davidmenendez** [carbon-design-system/carbon-ai-chat#1279]
### High severity

**SL No: 3 + 4 - video in grid response controls [demo]**

> While navigating in browse mode using JAWS, focus moves to elements associated with video controls that are visually hidden or not available on the screen. JAWS announces these elements as interactive controls such as links and buttons. When users attempt to interact with them, some links open new pages, even though these controls are not visibly present or expected within the video player. This creates a mismatch between what is visually presented and what is exposed to assistive technologies.

> "Ensure that hidden or inactive controls are not exposed to screen readers. Specifically: Remove hidden elements from the accessibility tree using aria-hidden="true" or appropriate visibility handling.
Ensure that non-visible controls are not focusable (e.g., avoid tabindex="0" on hidden elements)."

> JAWS focus gets trapped inside the “More options” combo box on the video player after it is activated, preventing users from navigating away in browse mode.

will need to verify this isn't just an issue with the vimeo layout specifically, which is beyond our control.

### Medium severity

**SL No: 1 - grid heading hierarchy [demo]**

the headings in the grid response like `WidthOptions.SMALL` should be h3 instead of h4

**SL No: 2 - jaws repeating select message after selecting grid option [demo]**

i am currently unable to replicate this. will need to follow up with accessibility team.

**SL No: 5 - grid response not in a landmark region [demo]**

> "Ensure that the grid response is properly structured within an appropriate landmark. Specifically: Wrap the grid response content within a semantic landmark element such as <main>, <section>, or a container with role="region". Provide an accessible name for the landmark using aria-label or aria-labelledby."

**SL No: 6 - heading in human agent card heading hierarchy [demo]**

`Live agent support` is styled like a heading but isn't using a semantic tag

**SL No: 7 - human agent disconnect focus [demo]**

when you click disconnect when connected with a human agent a confirmation dialog opens up, but the first interactive element isn't being focused

**Comment by davidmenendez** [comment]
i've determined that because these are all demo related issues that there's nothing for us to prioritize here. closing as completed.
